### PR TITLE
Adding workspace selection component

### PR DESCRIPTION
**Pull request by adamread**
This adds the component and page used to select the workspace used for the tests.

The accounts are fetched before the redirect to the new page to avoid showing the "no accounts" message while waiting on the async fetch to finish.

The workflow is vertical, because that's the easiest way to do it. A horizontal workflow for wider viewports could be added at a later date if desired.

### CODE REVIEW

**Comment by adamread** [comment]
This should be reviewed after the first tests' PR. It will simplify the review.

File: src/account_list.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    header {
+      background-color: #AECBFA;
+      color: white;
+      text-align: center;
+      margin-bottom: 5%;
+      padding: 3vh;
+    }
+
```
Comment (notmariazoe) [inline src/account_list.html:24]: Very NIT, and might be an issue with the editor, but the uneven spacing between blocks is mildly bothering me.

Comment (adamread) [reply src/account_list.html:15]: Do you mean the indentation? Likely my editor being silly, but I think I fixed it. 

@notmariazoe Can you check that that is what you're looking for, please?

Comment (notmariazoe) [reply src/account_list.html:15]: Potentially, there was a double blank space - Fixed now thanks

**Review by notmariazoe — APPROVED** [review]
LGTM

File: src/controllers/tagmanager-controller.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ const localStorage = window.localStorage;
  */
 export async function fetchAccounts() {
   await authorizedXhr('https://www.googleapis.com/tagmanager/v2/accounts').then(
-    (xhr: any) => {
+    xhr => {
```
[the remark this answers is not in the document]
Comment (adamread) [reply src/controllers/tagmanager-controller.ts:35]: I added the return type to authorizedXhr. It was in the comments, but that's not enough for typescript.

**Pull request merged**